packages/styles/components/Tooltip.module.scss

```diff
@@ -58,15 +58,27 @@ $arrow-size: 5px;
 
       &::after {
         border-color: var(--default-color) transparent transparent transparent;
-        left: 50%;
-        margin-left: (-$arrow-size);
         top: 100%;
       }
     }
 
     &.light {
-      .tooltipContent::after {
-        border-color: var(--white) transparent transparent transparent;
+      .tooltipContent {
+        &::after {
+          border-color: var(--white) transparent transparent transparent;
+        }
+
+        &::before {
+          border-color: var(--gray-2) transparent transparent transparent;
+          border-style: solid;
+          border-width: ($arrow-size + 2px);
+          content: '';
+          left: 42%;
+          position: absolute;
+          top: 127%;
+          transform: translateX(-50%);
+          transform: translateY(-50%);
```

Comment:
`transform: translateY(-50%);` will overwrite `transform: translateX(-50%);`, maybe we should use `transform: translate(-50%, -50%)`

Reply:
Done :D